docs/source/settings.rst

```diff
@@ -54,25 +74,45 @@ For example, the following suite will be executed every 30 minutes:
 SPIDERMON_SPIDER_CLOSE_MONITORS
 -------------------------------
 
+Default: ``[]``
+
 List of monitor suites to be executed when the spider closes.
 
 .. _SPIDERMON_SPIDER_CLOSE_EXPRESSION_MONITORS:
 
 SPIDERMON_SPIDER_CLOSE_EXPRESSION_MONITORS
 ------------------------------------------
 
-.. _SPIDERMON_SPIDER_OPEN_EXPRESSION_MONITORS:
+Default: ``[]``
 
-SPIDERMON_SPIDER_OPEN_EXPRESSION_MONITORS
------------------------------------------
+List of dictionaries, each dictionary stands for a MonitorSuite with one
```

Comment:
What about a single paragraph like this?

```
List of dictionaries describing :ref:`expression monitors <expression-monitors>` to run when a spider is closed.
```

A similar approach could be followed for the other related settings.